checksums.sh

```diff
@@ -15,18 +15,19 @@ then
 fi
 
 SUBJECT_FOLDER=$1
+QSUB_FILE=".qsub.out.*"
 
-echo "###########################################"
-echo "############ Checksum of docker image #####"
-echo "###########################################"
-
-grep "Digest: sha256:" .qsub.out.*
+count=`ls -a 2>/dev/null | grep $QSUB_FILE | wc -l`
+if [ $count != 0 ]
+then
```

Comment:
If the file is already existing in the docker images repo, would we get a checksum in the .qsub.out files ? If not then we need not stop I think. Thoughts ?